docs/projects/ad9213_evb/index.rst

```diff
@@ -0,0 +1,231 @@
+.. _ad9213_evb:
+
+AD9213 HDL project
+===============================================================================
+
+Overview
+-------------------------------------------------------------------------------
+
+The :adi:`AD9213-EVB <EVAL-AD9213>` reference design is a processor based 
+(e.g. Microblaze) embedded system.
+
+The design implements a high-speed receive chain using JESD204B.
+
+The receive chain transports the captured samples from ADC to the system
+memory (DDR). Before transferring the data to DDR the samples are stored
+in a buffer implemented on block rams from the FPGA fabric
+(:git-hdl:`util_adcfifo <library/util_adcfifo>`).
+
+All cores from the receive chain are programmable through
+an AXI-Lite interface.
+
+Supported boards
+-------------------------------------------------------------------------------
+
+-  :adi:`AD9213-EVB <EVAL-AD9213>`
+
+Supported devices
+-------------------------------------------------------------------------------
+
+-  :adi:`AD9213`
+
+Supported carriers
+-------------------------------------------------------------------------------
+
+.. list-table::
+   :widths: 35 35 30
+   :header-rows: 1
+
+   * - Evaluation board
+     - Carrier
+     - FMC slot
+   * - :adi:`AD9213-EVB <EVAL-AD9213>`
+     - :xilinx:`VCU118`
+     - FMC+
+
+Block design
+-------------------------------------------------------------------------------
+
+Block diagram
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The data path and clock domains are depicted in the below diagrams:
+
+.. image:: ad9213_jesd_diagram.svg
+   :width: 800
+   :align: center
+   :alt: AD9213-EVB JESD204B M=1 L=16 block diagram
+
+The Rx links (ADC Path) operate with the following parameters:
+
+-  Rx parameters: L=16, M=1, F=2, S=16, NP=16, N=16 
+-  Sample Rate: 10 GSPS
+-  Dual link: No
+-  RX_DEVICE_CLK: 312.25 MHz (Lane Rate/40)
+-  REF_CLK: 625 MHz (Lane Rate/20)
+-  JESD204B Lane Rate: 12.5 Gbps
+-  QPLL0
+
+.. warning::
+
+   ``Lane Rate = I/Q Sample Rate x M x N' x (10 \ 8) \ L``
```

Comment:
Perhaps use the math syntax
```suggestion
.. math::

   Lane Rate = \frac{I}{Q}*Sample Rate*{M*N'}{L}*\frac{10}{8}
```

if changing to it, also update please:
docs/projects/ad9081_fmca_ebz/index.rst
docs/projects/adrv9026/index.rst
docs/projects/adrv904x/index.rst

And perhaps even
projects/ad9081_fmca_ebz/a10soc/system_project.tcl
projects/ad9081_fmca_ebz/common/ad9081_fmca_ebz_qsys.tcl
projects/ad9081_fmca_ebz/s10soc/system_project.tc